GameWorld/View3D/Services/SceneSaving/Material/Strategies/PharaohWsModelStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GameWorld.Core.SceneNodes;
+using Shared.Core.Services;
+
+namespace GameWorld.Core.Services.SceneSaving.Material.Strategies
+{
+    internal class PharaohWsModelStrategy : IMaterialStrategy
+    {
+        private readonly WsModelGeneratorService _wsModelGeneratorService;
+
+        public MaterialStrategy StrategyId => MaterialStrategy.WsModel_Pharaoh;
+        public string Name => "Pharaoh";
+        public string Description => "Hopefully generates WsModel";
```

Comment:
The description should be more professional.
It also makes me fear that the PR has not been thoroughly tested?